Playground/python-openai-server/server.py

```
@@ -0,0 +1,306 @@
+    file: UploadFile = File(...),
+    model: str = Form(DEFAULT_STT),
+):
+    stt, lock = await _models().stt(model)
+    raw = await file.read()
+    try:
+        sample_rate, samples = decode_wav(raw)
+    except Exception as exc:  # noqa: BLE001
+        raise HTTPException(
+            status_code=400,
+            detail=f"could not decode audio (send 16 kHz mono WAV): {exc}",
+        )
```

Ruff (0.15.21)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)